tests/importer_module.py

```diff
@@ -1,19 +1,7 @@
-import pytest
-import json
-from fastapi.testclient import TestClient
-
-# This is a pretty delicate thing, any change could cause it to break
-# Is there a better way to make this work?
-import sys
-
-from thor.main import app
-
-client = TestClient(app)
+# importer_module.py
 
+import json
 
-print("working")
-# Question: Would it be better to put the expected outputs into a separate file,
-# to avoid making the code harder to read?
 expected_output_for_get_releases = {
```

Comment:
I thought we were going to get rid of any `expected_output` variables and only leverage actual json files stored in the `tests` folder. Should we leave this variable here?